### PR TITLE
New site banner

### PR DESCRIPTION
**Pull request by iamdamion**
Added override folder/file needed for banner and updated files to get banner redirecting to the new site active.

### CODE REVIEW

**Comment by iamdamion** [comment]
@bendhouseart Please review and merge banner addition. Will need to discuss text I'm sure, but format and banner are working.

File: mkdocs.yml [hunk 1]
```diff
@@ -22,11 +22,13 @@ nav:
 
 theme:
     name: material
+    custom_dir: docs/overrides # - allows use of the uverride/main.html file
```
Comment (bendhouseart) [inline mkdocs.yml:25]:
```suggestion
    custom_dir: docs/overrides # - allows use of the overrides/main.html file
```

**Review by bendhouseart — APPROVED** [review]
lgtm

File: docs/overrides/main.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      margin: 0
+    }
+  </style>
+  <div class="announcement">
+
+    <h2><b>THIS SITE IS BEING MERGED</b><br></h2>
+
+    <h3>
+      The BIDS group is currently finalizing a NEW main website that will merge our existing websites into one central, easy to navigate resource. <br>
+      The <b><a href="https://bids-website.readthedocs.io" target="_blank">New BIDS Wesbite</a> (https://bids-website.readthedocs.io)</b>
```
Comment (Remi-Gau) [inline docs/overrides/main.html:20]:
```suggestion
      The <b><a href="https://bids-website.readthedocs.io" target="_blank">New BIDS Website</a> (https://bids-website.readthedocs.io)</b>
```

**Comment by Remi-Gau** [comment]
Good to go!!!
Thanks @iamdamion

**Pull request merged**